Source/WebCore/platform/graphics/gstreamer/eme/WebKitThunderDecryptorGStreamer.cpp

```diff
@@ -84,21 +82,21 @@ static GRefPtr<GstCaps> createSinkPadTemplateCaps()
         return caps;
     }
 
-    for (int i = 0; cencEncryptionMediaTypes[i]; ++i) {
+    for (auto& mediaType : cencEncryptionMediaTypes) {
         gst_caps_append_structure(caps.get(), gst_structure_new("application/x-cenc", "original-media-type", G_TYPE_STRING,
-            cencEncryptionMediaTypes[i], nullptr));
+            mediaType.characters(), nullptr));
     }
     for (const auto& keySystem : supportedKeySystems) {
-        for (int i = 0; cencEncryptionMediaTypes[i]; ++i) {
+        for (auto& mediaType : cencEncryptionMediaTypes) {
```

Comment:
could this be 

```suggestion
        for (const auto& mediaType : cencEncryptionMediaTypes) {
```
or does the structure need a non-const pointer?

Reply:
const pointers work indeed, good catch!